datadm/backend.py

```diff
@@ -11,7 +11,7 @@
     local_available = False
 
 class StarcoderChat(guidance.llms.Transformers):
-    def __init__(self, model_path="HuggingFaceH4/starchat-alpha", **kwargs):
+    def __init__(self, model_path="HuggingFaceH4/starchat-beta", **kwargs):
```

Comment:
this will probably fail because I am pulling in 2 lines a specific revision. 

My suggestion to add starchat-beta would be to:

1. update the `revision` to be a kwarg, and default to it
2. update the `BackendLLMManager` object's `llms` to have a new entry `starchat-alpha-cuda` and `starchat-beta-cuda` as 2 separate entries. 
3. update the `load` step for each, eg. the `alpha` could be the same, and beta would be `StarcoderChat(model="HuggingFaceH4/starchat-beta", revision=None)` kinda thing

Reply:
Cool! 
Please see the proposed changes here: https://github.com/approximatelabs/datadm/pull/4/commits/a24ab875ac0bb0aa96a0c6820839558565d2c6f8

I couldn't fully test it since I'm currently stuck downloading the models from Huggingface and getting a read timeout everytime (I've only used the gpt3.5 option so far as the gpt-4 doesn't work for me as well).
Is there any easy option to download the model files manually? 
(I'm using the Docker image and saw they're saved as symlinks to blob directories but haven't digged deeper yet), 
or perhaps any easy option to add resuming capabilities when downloading from Huggingface?

Reply:
Not sure about downloading manually -- there are ways with git lfs, but i have never done it.

This worked for me, it "just worked" -- loaded the model, took about ~5 min first download, then when i came back, i restarted datadm and it loaded from cache and loaded up in ~20 seconds into my GPU, and worked!